source/libs/page-detect.ts

```diff
@@ -372,6 +372,16 @@ export const _isSingleTagPage = [
 	'https://github.com/sindresorhus/refined-github/releases/tag/0.2.1'
 ];
 
+export const isPulse = (): boolean => /^pulse/.test(getRepoPath()!);
+export const _isPulse = [
+	'https://github.com/sindresorhus/refined-github/pulse'
+];
+
+export const isActions = (): boolean => /^actions/.test(getRepoPath()!);
+export const _isActions = [
+	'https://github.com/sindresorhus/refined-github/actions'
```

Comment:
When adding test URLs, also add possible false positives, like /actions, sindre/actions, etc.

False positives have to be added in their respective groups, for example sindre/actions would be `isRepo`

Reply:
Your right.